cloud/blockstore/libs/storage/disk_agent/disk_agent_actor_stats.cpp

```diff
@@ -181,6 +181,7 @@ void TStatsActor::CheckDevicesHealth(const TActorContext& ctx)
             std::make_unique<TEvDiskAgent::TEvReadDeviceBlocksRequest>();
         auto& rec = request->Record;
         rec.MutableHeaders()->SetClientId(TString(CheckHealthClientId));
+        rec.MutableHeaders()->SetRequestId(ctx.Now().MicroSeconds());
```

Comment:
don't you want to use `SetRequestTimestamp` instead ? By the way why do you need this change ?